codacy/templates/secret.yaml

```diff
@@ -0,0 +1,68 @@
+apiVersion: v1
```

Comment:
This file defines all the queues rabbit must create, with the exception of the `crow` queue, which is created automatically by the component upon a restart from rabbit's side.

It is recommended as per the bitnami/rabbitmq chart documentation to inject these additional configurations through a .json file that comes from a secret.